design-system/uilib/src/components/dpdhl-pagination/dpdhl-pagination.svelte

```diff
@@ -0,0 +1,127 @@
+<script context="module" lang="ts">
+	export const ITEMS = {};
+</script>
+
+<script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import { get_current_component, onMount } from "svelte/internal";
+
+	export let count 		= 12;
+	export let maxpages 	= 10;
+	export let defaultpage 	= 1;
+	
+	$: selectedItem = defaultpage;
```

Comment:
does this need to be reactive?
It would mean every time the `defaultpage` (should be `defaultPage` with camelCase) changes the `selectedIdem` will be set to the same value